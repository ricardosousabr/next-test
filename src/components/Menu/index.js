export default function Menu() {
  return(<div>
    <ul>
    <li><a href="">Home</a></li>
    <li><a href="">Help</a></li>
    <li><a href="">Contact</a></li>
    <li><a href="">profile</a></li>
    </ul>
  </div>)
}