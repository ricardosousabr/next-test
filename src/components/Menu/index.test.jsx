import { render } from '@testing-library/react'
import Menu from './index'

it('renders homepage unchanged', () => {
  const { container } = render(<Menu />)
  expect(container).toMatchSnapshot()
})