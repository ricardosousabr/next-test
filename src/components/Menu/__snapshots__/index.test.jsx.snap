// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div>
    <ul>
      <li>
        <a
          href=""
        >
          Home
        </a>
      </li>
      <li>
        <a
          href=""
        >
          Help
        </a>
      </li>
      <li>
        <a
          href=""
        >
          Contact
        </a>
      </li>
      <li>
        <a
          href=""
        >
          profile
        </a>
      </li>
    </ul>
  </div>
</div>
`;
